include/exadg/utilities/hypercube_resolution_parameters.h

```diff
@@ -71,13 +71,13 @@ fill_resolutions_vector(
 {
   unsigned int l = 0, n_subdivisions_1d = 1;
 
-  double n_cells_min = (double)n_dofs_min / dofs_per_element;
-  double n_cells_max = (double)n_dofs_max / dofs_per_element;
+  types::global_dof_index n_cells_min = n_dofs_min / dofs_per_element;
+  types::global_dof_index n_cells_max = (n_dofs_max + dofs_per_element - 1) / dofs_per_element;
```

Comment:
I am currently not sure if this is correct. The user expects that `n_dofs_min <= n_dofs <= n_dofs_max` for all `n_dofs` realized in the throughput experiment. So we would have to ceil `n_cells_min` and floor `n_cells_max`?

Reply:
I agree, my formula did it the wrong way around (floor/ceil), including slightly too much.